models/articles.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../db/connection');
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query(`
+      SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .catch((err) => {
```

Comment:
We don't need this catch here. A promise chain only need one catch block, which should be at the end of the promise chain. Let the error go through to the catch in you controller and be piped into your error handling middleware instead